deployment/systemd/reader.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Reader app uberjar
+After=network.target
+
+[Service]
```

Comment:
The service should be run as a non-root user. This can be set using `User=`. https://www.freedesktop.org/software/systemd/man/systemd.exec.html#